wordpress-mobile/WordPress-iOS
# Follow Conversation Tooltip remove local feature flag

Fixes pbArwn-4oo-p2

This PR removes the local feature flag and adds unit tests.

To test:
1. Go to Reader
2. Into any post that is not already being followed
3. Validate if the tooltip/anchor appears
4. Feature Flag: Feature Highlight Tooltip must be removed from the feature flag list in Debug View.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Added unit tests for `FeatureHighlightStore`

PR submission checklist:
